software/wmla120_ansible/validate_firewall_setup.yml

```diff
@@ -2,61 +2,68 @@
 #If a firewall is enabled, the following default ports must be granted access on all management hosts for IBM Spectrum Conductor Deep Learning Impact: 9243, 9280, 5000, 5001, 27017, and 6379. If you change these ports after installation, make sure to update firewall rules accordingly.
 #
 
-- name: Setting Variables
-  set_fact:
-    port_9243: '9243/tcp'
-    port_9080: '9080/tcp'
-    port_5000: '5000/tcp'
-    port_5001: '5001/tcp'
-    port_27017: '27017/tcp'
-    port_6379: '6379/tcp'
+- name: Include vars of envs_spectrum_conductor_dli.yml into the 'ports' variable
+  include_vars:
+    file: envs_spectrum_conductor_dli.yml
+    name: ports
 
-- name: List all host firewall data
-  shell: "firewall-cmd --list-all"
+- name: DEBUG - List imported variables
+  debug:
+    msg: "{{ ports }}"
+
+- name: Check status of firewall
+  shell: "firewall-cmd --state"
   delegate_to: localhost
```

Comment:
delegate_to: localhost needs to be removed.  We need this run on the client nodes.

Reply:
okay, I have removed. Thanks!